lib/heroku_rails_deploy/deployer.rb

```diff
@@ -72,7 +72,7 @@ def push_code(app_name, revision)
     end
 
     def git_branch_name
-      run_command('git rev-parse --abbrev-ref HEAD')
+      `git rev-parse --abbrev-ref HEAD`.strip
```

Comment:
Should we rescue with something arbitrary `'not-a-branch'` or rescue nil and if it's nil we skip the check?